CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 # CHANGELOG
 
-## Pending
+## 1.7.0
+
+- Added support for double tapping, long pressing and two finger tapping, with default implementations for any Elements that conform to Tappable. (issue #88)
+- Added Swift Package Manager support
+- Added a `Refreshable` protocol so for each element/screen that conforms to protocol `refresh` function can be defined 
+- Added a `DatePicker` element. (issue #55)
 
```

Comment:
`value` can now be accessed directly from elements

Reply:
Is this okay, @SamDKBerry ?

Reply:
Looks good